### ISSUE TITLE
Missing labels

### ISSUE THREAD

**Issue by lnacquaroli** [baibai25/chestnut-seaurchin-classifier#2]
Hello there, I have been trying to use the demo as posted in the README and it was successful. I get the bounding box around the urchins' pictures, but the labels are missing, it shows black patches instead.

One thing I noticed is that the `data/labels` are missing, but not sure if it is related. I get a lot of this:

```
STB Reason: can't fopen
Cannot load image "data/labels/100_7.png"
```

Any ideas? Thanks for the repo. We are trying to build a detector of urchins too and found this one really useful.

**Comment by baibai25** [comment]
@lnacquaroli 
Thank you for the message! I've been away from this project for a long time so I need some time to remember the details. 
Could you tell me details of your environment and library version? I will check it, but it will take some time due to my job.

**Comment by lnacquaroli** [comment]
Hi, thanks for the answer. No worries.

I am actually using the google-colab environment, with python 3.7.12 in it. I haven't loaded any other special library, except to see the predicited image.

This is basically what I am doing:

```python
from google.colab.patches import cv2_imshow
import cv2

# Build the darknet
!git clone https://github.com/AlexeyAB/darknet.git
cd darknet
!make

cd ..

# Download the weights
!wget --no-check-certificate 'https://drive.google.com/u/0/uc?export=download&confirm=saQD&id=1Yoq3Nwx72zXTtANvEqp_aNCvoo3wtk_A' -O 'unikuri_best.weights'

!git clone https://github.com/baibai25/chestnut-seaurchin-classifier

!darknet/./darknet detector test cfg/unikuri.data cfg/unikuri.cfg unikuri_best.weights sea_urchin.jpeg
```

Where `sea_urchin.jpeg` is a random image from the web.

**Comment by baibai25** [comment]
Thank you for share your information!

I checked with google-colab and got the same errors.
It seems to require a dataset for prediction, even though it uses a trained model. I didn't know about it...

Unfortunately, I've already discarded the dataset. (I thought there was no demand for it lol)

However, the weights file can probably be converted to other format such as tensorflow or pytorch. I will continue investigations about that.

**Comment by lnacquaroli** [comment]
Yes, it is weird that you need the dataset for the prediction I guess. Maybe the darknet changed a bit since you made your tests.

I will continue to check, otherwise, I might give it a try to train a scrapped dataset.

Thanks for looking into it.

**Comment by lnacquaroli** [comment]
@baibai25

So, I finally got it to work. It was my mistake indeed since I was changing directories incorrectly when executing the `darknet detector`.

Now it's working just fine. It would be nice to know what data you used to train the network, or did you scrape it all? It is a shame you don't have it anyways because the labels are lost now.

Thanks again for your time.
